src/App.js

```diff
@@ -1,28 +1,60 @@
-import logo from './logo.svg';
-import './App.css';
-import { useFeaturedBanners } from './utils/hooks/useFeaturedBanners';
+import { useEffect, useState } from 'react';
+import Footer from './components/common/Footer';
+import Header from './components/common/Header';
+import Flex from './components/common/Flex';
+import Slider from './components/common/Slider';
+import ImageContainer from './components/common/Image';
+import Card from './components/common/Card';
+import { results as productCategoriesResults } from './utils/product-categories.json';
+import { results as featuredBannersResults } from './utils/featured-banners.json';
+import { results as featuredProductsResults } from './utils/featured-products.json';
 
 function App() {
-  const { data, isLoading } = useFeaturedBanners();
-  console.log(data, isLoading);
-
+  const [productsCategories, setProductsCategories] = useState([]);
+  const [featuredBanners, setFeaturedBanners] = useState([]);
+  const [featuredProducts, setFeaturedProducts] = useState([]);
+  
+  useEffect(() => {
+    setProductsCategories(productCategoriesResults);
+    setFeaturedBanners(featuredBannersResults);
+    setFeaturedProducts(featuredProductsResults);
+  },[])
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Header/>
+      <Flex>
+        <Slider items={featuredBanners} />
+      </Flex>
+      <hr/>
+      <Flex wrap={"wrap"} justifyContent={"space-around"} alignItems={"flex-start"}>
```

Comment:
<!-- Thoughts represent an idea that popped up from reviewing. These comments are non-blocking by nature, but they are extremely valuable and can lead to more focused initiatives and mentoring opportunities.  -->
**thought:** ‏This is a great way of using styled components, congrats :)